src/Module.Server/Common/CrpgTeamSelectComponent.cs

```diff
@@ -343,13 +343,13 @@ private float ComputeRatingWeight(CrpgUser user)
         var rating = user.Character.Rating;
         float regionPenalty = CrpgRatingHelper.ComputeRegionRatingPenalty(user.Region);
         // https://www.desmos.com/calculator/snynzhhoay
-        return 6E-8f * (float)Math.Pow(rating.Value - 2 * rating.Deviation, 3f) * regionPenalty;
+        return 6E-8f * (float)Math.Pow(rating.Value - 2 * rating.Deviation, 3.98f) * regionPenalty;
     }
 
     private float ComputeEquippedItemsWeight(IList<CrpgEquippedItem> equippedItems)
     {
         float itemsPrice = ComputeEquippedItemsPrice(equippedItems);
-        return 1f + itemsPrice / 50_000f;
+        return 1f + itemsPrice / 500_000f;

```

Comment:
Does it mean the equipment has a very low impact on the round outcome? Since some builds use low cost stuff and cannot be considered worse than some using high cost stuff I'm wondering if we should not remove the equipment weight all together.

Reply:
50000/500_000 is 10% . Also we played with Putting a Min(itemsPrice, maxItemPrice) , and we realized that it made thing worse. There was one dude with a 113k build that had a decent impact on games. This will have a 20% penalty. We also tested no equipment weight and it was just worse in all ways. So no , this is still needed.